index.html

```diff
@@ -66,73 +66,22 @@ <h2>Learning that gets you</h2>
             <h2>A broad selection of courses</h2>
             <p>Choose from 185,000 online video courses with new additions published every month</p>
             <ul class="courses_list">
-                <li>Python</li>
-                <li>Excel</li>
-                <li>Web Development</li>
-                <li>JavaScript</li>
-                <li>Data Science</li>
-                <li>AWS Certification</li>
-                <li>Drawing</li>
+                <li id="c_python">Python</li>
+                <li id="c_excel">Excel</li>
+                <li id="c_web">Web Development</li>
+                <li id="c_js">JavaScript</li>
+                <li id="c_data">Data Science</li>
+                <li id="c_aws">AWS Certification</li>
+                <li id="c_draw">Drawing</li>
```

Comment:
You can also load this dynamically from the javascript, this way in the future if you want to make any change you will only update one place in the code and also can update the categories from the backend.